explore/src/main/scala/explore/common/ScienceQueries.scala

```diff
@@ -117,7 +93,9 @@ object ScienceQueries:
                  .orUnassign
           _ <- SpectroscopyScienceRequirementsInput.resolution         := op.resolution.orUnassign
           _ <-
-            SpectroscopyScienceRequirementsInput.exposureTimeMode := op.exposureTimeMode.toInput.orUnassign
+            SpectroscopyScienceRequirementsInput.exposureTimeMode := op.exposureTimeMode
+              .map(_.toInput)
+              .orIgnore
```

Comment:
You switched from `orUnassign` to `orIgnore`. I just want to make sure that is intentional and not an autocomplete error. 😆

Reply:
Good question. I think `orUnassign` though I can't see how the UI would set it to null
I'll change it nonetheless